properties.schema

```diff
@@ -3,6 +3,26 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "id": "http://jsonschema.net",
   "required": false,
+  "globals": {
+    "resumeButtonLabel": {
```

Comment:
I think we normally call these button text rather than button label.

Reply:
Will apply